prediction/src/algorithms/segment/src/data_generation.py

```diff
@@ -0,0 +1,63 @@
+import glob
+import os
+
+import numpy as np
+import pylidc as pl
+
+
+def get_dicom_paths(in_docker=True):
+    """Return DICOM paths to all LIDC directories
+    e.g. ['../images_full/LIDC-IDRI-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178/' \
+          '1.3.6.1.4.1.14519.5.2.1.6279.6001.179049373636438705059720603192']"""
+    if in_docker:
+        return glob.glob("../images_full/LIDC-IDRI-*/**/**")
+    else:
```

Comment:
Why the ``else`` if we have unconditional return flow on the previous line?

Reply:
If the method is not executed from within the Docker environment, the LIDC images aren't at /images_full but at test/assets/... . If you want to train the model, it would be recommended to do that outside of the Docker environment as it lacks of CUDA support. To also be able to use the GPU from within Docker, we'd be supposed to use [nvidia-docker](https://github.com/NVIDIA/nvidia-docker). Thus, I thought it would make sense to have a switch in the method to also get correct DICOM paths if you're outside of the Docker environment, inside of the prediction package and trying to train your model. Does that make sense?

Reply:
I think what @lamby is saying is that you could refactor to something like this

```python 
    if in_docker:
        return glob.glob("../images_full/LIDC-IDRI-*/**/**")

    return glob.glob("../tests/assets/test_image_data/full/LIDC-IDRI-*/**/**")
```

or (even better) just use one return

```python
    if in_docker:
        path = glob.glob("../images_full/LIDC-IDRI-*/**/**")
    else:
        path = glob.glob("../tests/assets/test_image_data/full/LIDC-IDRI-*/**/**")

    return path
```

Reply:
ack

Reply:
Done in 28660bf681dad4acae8e20ee15433d8a48884384